pydantic_ai_slim/pydantic_ai/models/google.py

```diff
@@ -267,7 +267,7 @@ async def count_tokens(
             messages, model_settings, model_request_parameters
         )
 
-        # Annoyingly, the type of `GenerateContentConfigDict.get` is "partially `Unknown`" because `response_schema` includes `typing._UnionGenericAlias`,
+        # Annoyingly, the type of `GenerateContentConfigDict.get` is "partially `Unknown`" because `response_json_schema` includes `typing._UnionGenericAlias`,
```

Comment:
I don't think this applies to `response_json_schema` anymore, so we may be able to clean this code up a bit